internal/jujuapi/cloud.go

```diff
@@ -481,21 +481,20 @@ func (r *controllerRoot) UpdateCloud(ctx context.Context, args jujuparams.Update
 	results := jujuparams.ErrorResults{
 		Results: make([]jujuparams.ErrorResult, len(args.Clouds)),
 	}
-	for i := range args.Clouds {
-		err := r.updateCloud()
+
+	for i, cloud := range args.Clouds {
+		ct, err := names.ParseCloudTag("cloud-" + cloud.Name)
```

Comment:
I always forget the best way of creating tags, is this it? Using "cloud-" isn't amazing but I can't see an alternate way.

Reply:
what about `cloud.Tag()`?